Configuration/TCA/tx_academicpersons_domain_model_organisational_unit.php

```diff
@@ -121,26 +116,6 @@
                 'type' => 'text',
             ],
         ],
-        'valid_from' => [
-            'label' => $ll('columns.valid_from.label'),
-            'l10n_mode' => 'exclude',
-            'l10n_display' => 'defaultAsReadonly',
-            'config' => [
-                'type' => 'input',
-                'renderType' => 'inputDateTime',
-                'eval' => 'date',
-            ],
-        ],
-        'valid_to' => [
```

Comment:
Property, getter and setter is not removed from corresponding `Fgtclb\AcademicPersons\Domain\Model\OrganisationalUnit` Domain
Model.

Which could be breaking. I would emphasize this using

```
[!!!][TASK]
```

prefix for commit and pull-request.

Reply:
valid_to is kept.